extensions/aerospace/src/showConfig.tsx

```diff
@@ -0,0 +1,17 @@
+import { Detail } from "@raycast/api";
+import TOML from "@iarna/toml";
+import { getConfig } from "./utils/config";
+
+export default function checkConfig() {
```

Comment:
I'm curious what is the purpose of "Show Aerospace Config" command?

When the config lines are all squashed together it doesn't look useful

<img width="750" alt="image" src="https://github.com/raycast/extensions/assets/20517828/0ffca8ef-dd6f-4642-96d1-f4cc2a1a1982">

Reply:
This can definitely be improved visually a lot i wanted to see the config file at a glance to see if I had any parsing errors and if users have it this would print out the error nicely

Reply:
Thinking on it, I was actually going for validation of the parsing logic so i can just print out whether extension parsed the config successfully or not and chabge the name to validate

Reply:
I can introduce a plumbing CLI command to print out all parsing and semantic errors to stdout so this CLI command can be used instead

> and chabge the name to validate

Agree, I have the following suggestions:
- Validate AeroSpace config
- Check AeroSpace config

Reply:
Great, I want to help on the plumbing side as well but I don't now Swift but if you can give some pointers, I can try to contribute this week!